tensorflow_probability/python/experimental/mcmc/with_reductions_test.py

```diff
@@ -35,11 +35,12 @@
 class TestReducer(tfp.experimental.mcmc.Reducer):
   """Simple Reducer that just keeps track of the last sample"""
 
-  def initialize(self, initial_chain_state=None, initial_kernel_results=None):
-    return None
+  def initialize(self, initial_chain_state, initial_kernel_results=None):
+    return tf.zeros(tf.convert_to_tensor(initial_chain_state).shape)
```

Comment:
Use tf.zeros_like here, instead of indirecting through the shape.  It's more robust.